src/test/scala/uk/gov/hmrc/ct/computations/CP48Spec.scala

```diff
@@ -18,15 +18,15 @@ class CP48Spec extends WordSpec with MockitoSugar with Matchers {
   "CP48" should {
     "not be valid if it is None and CP29 is not" in {
       when(boxRetriever.cp29()).thenReturn(CP29(Some(1)))
-      CP48(None).validate(boxRetriever) shouldBe Set(CtValidation(Some("CP48"), "error.CP48.must.equal.CP29"))
+      CP48(None).validate(boxRetriever) shouldBe Set(CtValidation(Some("CP48"), "error.CP48.must.equal.CP29", Some(List("1"))))
```

Comment:
Where you have 3 lines here that are almost exactly the same. Could you simplify this at all?